ord_schema/proto/reaction.proto

```diff
@@ -465,7 +465,7 @@ message ReactionConditions {
   ElectrochemistryConditions electrochemistry = 5;
   FlowConditions flow = 6;
   optional bool reflux = 7;
-  optional float pH = 8;
+  optional float ph = 8;
```

Comment:
Not that it matters a ton, but why do this?

Reply:
@connorcoley and I talked about this offline; it's for consistency with `target_ph` in the workups.

Reply:
Got it